src/posix/platform/resolver.hpp

```diff
@@ -64,6 +64,14 @@ class Resolver : public Logger<Resolver>
      */
     void Setup(void);
 
+    /**
+     * Indicates whether an upstream DNS server is available for queries.
+     *
+     * @retval TRUE   An upstream DNS server is available.
+     * @retval FALSE  No upstream DNS server is currently available.
```

Comment:
```suggestion
     * @retval FALSE  An upstream DNS server is not available.
```

To align with each other and be consistent with wording from the platform API